packages/morpho-ts/README.md

```diff
@@ -47,27 +47,45 @@ Here are all the available formatters:
 - [**Short Formatter**](#short-formatter): Formats the value in a short notation with units (e.g., `k`, `M`).
 - [**Percent Formatter**](#percent-formatter): Formats the value as a percentage.
 
-### Usage
+#### Usage
```

Comment:
I know it's not indented correctly, but h4 is not that visible hence why I kept h3
Wdyt?

Reply:
I don't care much tbh